pyro/contrib/autoguide/__init__.py

```diff
@@ -63,10 +63,14 @@ class AutoGuide(object):
 
     Auto guides can be used individually or combined in an
     :class:`AutoGuideList` object.
+
+    :param callable model: A pyro model.
+    :param str name: A name that will be prefixed to all param internal sites.
     """
-    def __init__(self, model):
+    def __init__(self, model, name="auto"):
         self.master = None
         self.model = model
+        self.name = name
```

Comment:
nit: i prefer a name like `prefix` instead of `name` since this is prepended to the param names